src/domain/landingPage/landingPageHero/LandingPageHero.tsx

```diff
@@ -32,6 +34,77 @@ const LandingPageHero: React.FC<Props> = ({ landingPage }) => {
     window.open(buttonUrl || '', '_self');
   };
 
+  const width = React.useMemo(() => {
+    const getTextMaxWidth = (): number | undefined => {
+      switch (breakpoint) {
+        case 'xs':
+        case 'sm':
+          return undefined;
+        case 'md':
+          return 400;
+        case 'lg':
+        case 'xlg':
+          return 560;
+      }
+    };
+
+    const getTextFontSize = (): number => {
+      switch (breakpoint) {
+        case 'xs':
+        case 'sm':
+        case 'md':
+          return 52;
+        case 'lg':
+        case 'xlg':
+          return 80;
+      }
+    };
+
+    const maxTextWidth = getTextMaxWidth();
+
+    if (!maxTextWidth || !title) {
+      return undefined;
```

Comment:
returning undefined explicitly is redundant

Reply:
I decided to return null, if the maxTextWidth or title are empty. And moved that return statement to one place